doc/gpo/README.md

```diff
@@ -44,6 +44,22 @@ If this setting is not configured, experimentation is allowed.
 
 ### Installer and Updates
 
+#### Disable per-machine installation
```

Comment:
What use case should someone have?

Reply:
I was hoping for your suggestion here whether this is needed or not? :)

Reply:
I don't think so. If I deploy something as admin and want to control the Installation I need to prevent users from doing things that I approve. For example installing something as user without admin permission.

But I don't the any use case for blocking machine installation. And btw. Edge has only a block user install gpo too.

Reply:
OK. Makes sense. I'll remove it and if someone asks for it I can bring it back

Reply:
The only use case someone can have is setting both to block the app completely from installing. - Don't think that we want this. 🙈